st2client/st2client/commands/action.py

```diff
@@ -1269,7 +1266,7 @@ def __init__(self, resource, *args, **kwargs):
                                  help='Name of the workflow tasks to re-run.')
         self.parser.add_argument('--no-reset', dest='no_reset', nargs='*',
                                  help='Name of the with-items tasks to not reset. This only '
-                                      'applies to Mistral workflows. By default, all iterations '
+                                      'applies to Orquesta workflows. By default, all iterations '
```

Comment:
Option looks to have always been supported for Orquesta - it is in the documents and we had a unit-test, but the parser just referred to Mistral - so think it should have previously said both - so adjusted to Orquesta.